src/content/guides/prepare-azure-account-for-guest-clusters/index.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "Prepare an Azure subscription to run Giant Swarm guest clusters"
+description = "This guide will walk you through all necessary steps to set up an Azure subscription with approriate role definition and service principal for operating Giant Swarm guest clusters."
+date = "2018-08-09"
+type = "page"
+weight = 100
+tags = ["tutorial"]
++++
+
+# Prepare an Azure subscription to run Giant Swarm guest clusters
+
+In a Giant Swarm installation the guest clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription from the host cluster. This gives great flexibility depending on the requirements and the use case. For example, it allows the host cluster to be running in an Azure subscription owned by Giant Swarm, while guest clusters operate in different Azure subscriptions each, depending on a customer’s department using them.
+Giant Swarm operates guest clusters using a service called `azure-operator` which runs on the host cluster.
```

Comment:
`allows the host cluster to be running in an Azure subscription owned by Giant Swarm` - I think it's better to say it allows the host cluster to run in one subscription while ...

The host cluster doesn't have to run in our subscription.